vllm/attention/ops/paged_attn.py

```diff
@@ -34,7 +34,7 @@ class PagedAttention:
 
     @staticmethod
     def get_supported_head_sizes() -> List[int]:
-        return [64, 80, 96, 112, 120, 128, 192, 256]
+        return [32, 64, 80, 96, 112, 120, 128, 192, 256]
```

Comment:
TODO: It's strange that just adding another head size here makes the code run. Perhaps this is actually a silent failure and the actual kernel has to be added somewhere.